ssl.go

```diff
@@ -64,13 +68,13 @@ func ssl(o values) func(net.Conn) net.Conn {
 // in the user's home directory. The configured files must exist and have
 // the correct permissions.
 func sslClientCertificates(tlsConf *tls.Config, o values) {
-	sslkey := o.Get("sslkey")
-	sslcert := o.Get("sslcert")
+	sslkey, sslkeyOK := o["sslkey"]
+	sslcert, sslcertOK := o["sslcert"]
 
 	var cinfo, kinfo os.FileInfo
 	var err error
 
-	if sslcert != "" && sslkey != "" {
+	if sslkeyOK && sslcertOK {
```

Comment:
Fixed this, I think, though I think tests are going to break and I've not yet explicitly tested the empty string case.